src/Git/Git.cpp

```diff
@@ -2149,6 +2149,27 @@ CString CGit::GetGitSystemConfig() const
 
 BOOL CGit::CheckCleanWorkTree(bool stagedOk /* false */)
 {
+	if (m_IsUseLibGit2)
+	{
+		git_status_options statusopt = GIT_STATUS_OPTIONS_INIT;
+		git_status_list *status;
+		BOOL isClean = FALSE;
+		int errCode;
+
+		statusopt.show = GIT_STATUS_SHOW_WORKDIR_ONLY;
+		statusopt.flags = GIT_STATUS_OPT_UPDATE_INDEX | GIT_STATUS_OPT_EXCLUDE_SUBMODULES;
+
+		errCode = git_status_list_new(&status, g_Git.GetGitRepository(), &statusopt);
+
+		if (errCode == 0 && status)

```

Comment:
This check is problematic. status will newer be null, as it's never initialized.